spec/models/project_spec.rb

```diff
@@ -33,6 +33,11 @@
       expect(subject).to_not be_valid
     end
 
+    it 'should throw error for incomplete github url' do
+      subject.github_url = 'https://github.com/edx'
+      expect{ subject.github_repo_name }.to raise_error
```

Comment:
should avoid raise_error without specifics - ideally add exception type or error message